system/lib/wasmfs/wasmfs.cpp

```diff
@@ -63,9 +64,39 @@ extern "C" void wasmfs_flush(void) {
   // Flush musl libc streams.
   fflush(0);
 
-  // Flush our own streams. TODO: flush all backends.
+  // Flush our own streams.
   (void)SpecialFiles::getStdout()->locked().flush();
   (void)SpecialFiles::getStderr()->locked().flush();
+
+  std::stack<std::shared_ptr<Directory>> toFlush;
```

Comment:
Does it affect code size to use a vector here? I am hoping it might since we use vectors everywhere else, so the code might end up shared.

Reply:
Vectors saved around 1700 bytes!

Reply:
Nice!